packages/dcos-integration-test/extra/conftest.py

```diff
@@ -31,6 +31,13 @@ def pytest_collection_modifyitems(session, config, items):
     items[:] = new_items + last_items
 
 
+@pytest.fixture(autouse=True)
+def clean_marathon_state(dcos_api_session):
+    dcos_api_session.marathon.purge()
+    yield
+    dcos_api_session.marathon.purge()
+
```

Comment:
(Why) can we be sure that this will always be the "outest" fixture to be applied? I'd appreciate a post-review comment on this :).